helm/charts/kratos/values.yaml

```diff
@@ -187,6 +189,25 @@ kratos:
 
 # -- Configuration options for the k8s deployment
 deployment:
+  ## -- initContainer securityContext for hydra & migration init
+  initContainerSecurityContext: {}
+
+  ## -- pod securityContext for hydra & migration init
+  podSecurityContext: {}
+
+  ## -- container securityContext for hydra & migration init
+  securityContext:
+    capabilities:
```

Comment:
some alphabetical sorting would be nice here :)

Reply:
I copied this from Hydra - I think it makes sense to have the same ordering.

Reply:
The order will be alphabetically in the applied k8s resource anyway so would be nice it could be the same in the values as in the resource.